src/main/java/cart/data/MemberJdbcRepository.java

```diff
@@ -0,0 +1,54 @@
+package cart.data;
+
+import cart.data.entity.CartMember;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class MemberJdbcRepository implements MemberRepository {
+
+    private SimpleJdbcInsert simpleJdbcInsert;
+
+    public MemberJdbcRepository(DataSource dataSource) {
+        this.simpleJdbcInsert = new SimpleJdbcInsert(dataSource)
+                .withTableName("CART_MEMBER");
+    }
+
+    @Override
+    public List<CartMember> getMembers() {
+        String sql = "select * from CART_MEMBER";
+        JdbcTemplate jdbcTemplate = simpleJdbcInsert.getJdbcTemplate();
+
+        return jdbcTemplate.query(sql, carMemberRowMapper());
+    }
+
+    @Override
+    public Optional<CartMember> getMember(String email, String password) {
+        JdbcTemplate jdbcTemplate = simpleJdbcInsert.getJdbcTemplate();
+        String sql = "select * " +
+                "from CART_MEMBER " +
+                "where member_email = ? and member_password = ?";
+        CartMember cartMember;
+        try {
+            cartMember = jdbcTemplate.queryForObject(sql, carMemberRowMapper(), email, password);
+        } catch (Exception e) {
+            return Optional.empty();
+        };
+        return Optional.ofNullable(cartMember);
```

Comment:
CartMember 변수를 할당하는 과정이 필요하지 않아 보여요!
변수를 할당하지 않고 각각의 상황에 바로 반환하는 것은 어떻게 생각하시나요?

```suggestion
        try {
            return Optional.ofNullable(jdbcTemplate.queryForObject(sql, carMemberRowMapper(), email, password));
        } catch (Exception e) {
            return Optional.empty();
        };
```